src/examples/primitive_constraint_system.ts

```diff
@@ -79,6 +79,14 @@ const BitwiseMock = {
     Gadgets.xor(a, b, 48);
     Gadgets.xor(a, b, 64);
   },
+
+  not() {
```

Comment:
we should add the unchecked variant as well, either as its own method or directly in the `not` method